app/jobs/event/apply_fee_waiver_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Event
+  class ApplyFeeWaiverJob < ApplicationJob
+    queue_as :low
+
+    DATE_LOCK = Date.new(2025, 11, 1)
+
+    def perform
+      Event.find_each do |event|
+        process_event(event)
+      end
+    end
+
+    private
+
+    def process_event(event)
```

Comment:
This is okay, we need to be mindful of Standard organizations with a fee waiver being later changed to a "permanent" custom plan (e.g. HackClubHQ). This job would kick them back to Standard without the fee waiver